README.md

```diff
@@ -33,9 +33,13 @@ These instructions will get you a copy of the project up and running on your loc
 
 * [Java 11](https://openjdk.java.net/projects/jdk/11/) or higher
 * [Apache Maven 3.5](https://maven.apache.org/download.cgi) or higher
-* [Lombok 1.18.16](https://projectlombok.org/download) or higher. Please check the [requirements](https://projectlombok.org/setup/overview) for your IDE 
```

Comment:
Isn't this **still** a requirement for the project to be able to build, run, develop, …?

Reply:
Well, it's defined in the pom: Maven will handle the dependency for us, and as long as your IDE can build the maven project, one shouldn't have to need to install lombok manually (which is how I interpret this section).